README.md

```diff
@@ -137,6 +137,23 @@ The kernel has to be built with the following configuration:
 - `CONFIG_FTRACE=y`
 - `CONFIG_SCHED_CLASS_EXT=y`
 
+It's also recommended that you also include the following Kconfig options:
+- `CONFIG_BPF_JIT_ALWAYS_ON=y`
+- `CONFIG_BPF_JIT_DEFAULT_ON=y`
+- `CONFIG_KPROBES=y`
+- `CONFIG_PAHOLE_HAS_SPLIT_BTF=y`
+- `CONFIG_PAHOLE_HAS_BTF_TAG=y`
+
+A recommended config fragment is available in [`scx/sched_ext.config`](./sched_ext.config).
+It includes all required and recommended options for using `sched_ext`.
+You can append its contents to your kernel `.config` file to enable the necessary features.
+
+To apply it:
```

Comment:
I would drop this "To apply it" section, people that are willing to recompile the kernel likely know how to modify their config.